test/powershell/Modules/Microsoft.PowerShell.Management/FileSystem.Tests.ps1

```diff
@@ -246,6 +246,171 @@ Describe "Basic FileSystem Provider Tests" -Tags "CI" {
     }
 }
 
+Describe "Hard link and symbolic link tests" -Tags "CI", "RequireAdminOnWindows" {
```

Comment:
Added a test using the -Recurse parameter when removing a symlink to a directory